src/render/SDL_sysrender.h

```diff
@@ -366,6 +366,10 @@ extern void *SDL_AllocateRenderVertices(SDL_Renderer *renderer, size_t numbytes,
 // Let the video subsystem destroy a renderer without making its pointer invalid.
 extern void SDL_DestroyRendererWithoutFreeing(SDL_Renderer *renderer);
 
+// We must expose this function for manual control over renderer command buffer
```

Comment:
I didn't find a better way to expose some driver-specific functions aside from declaring them in sysrender.h